src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,78 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    private final static int LOTTO_BOUND_NUMBER = 45;
+    private final static int LOTTO_NUMBER_LIMIT = 6;
+
+    List<LottoNumber> lottoNumbers = new ArrayList<>();
+
+    public LottoTicket(Integer... lottoNumbers) {
+        this(toLottoNumbers(lottoNumbers));
+    }
+
+    private static List<LottoNumber> toLottoNumbers(Integer... numbers) {
+        List<LottoNumber> lottoNumbers = new ArrayList<>();
+        for (Integer number : numbers) {
+            lottoNumbers.add(new LottoNumber(number));
+        }
+        return lottoNumbers;
+    }
+
+    public LottoTicket(List<LottoNumber> lottoNumbers) {
+        validate(lottoNumbers);
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    private void validate(List<LottoNumber> lottoNumbers) {
+        if (lottoNumbers.size() > LOTTO_NUMBER_LIMIT) {
+            throw new IllegalStateException("로또의 숫자의 갯수 제한은 6개입니다.");
+        }
+    }
+
+    public static LottoTicket auto() {
+
+        Set<Integer> randomNumber = new HashSet<>();
+        Random rand = new Random();
+        getNotDuplicatedNumber(randomNumber, rand);
+
+        List<Integer> numbers = new ArrayList<>(randomNumber);
+        Collections.sort(numbers);
+
+        List<LottoNumber> lottoNumbers = createLottoNumbers(numbers);
+
+        return new LottoTicket(lottoNumbers);
```

Comment:
```suggestion
        List<Integer> numbers = getNotDuplicatedNumber();
        Collections.sort(numbers);
        List<LottoNumber> lottoNumbers = createLottoNumbers(numbers);
        return new LottoTicket(lottoNumbers);
```
메소드에 불필요한 인자가 들어가는 것 같아요
같은 기능인데 이렇게 만들어볼 수 없을까요?